### PR TITLE
docs: increase logo resolution

### PR DESCRIPTION
**Pull request by soobinrho**
Fixes [N/A]

**Changes proposed in this pull request:**
README.md is using the logo with the width of 400px, but the logo is only 298px, so it looks blurry. Here, it's being replaced by a higher resolution version. I downloaded the .svg version of the logo from
https://github.com/flarum/framework/blob/main/framework/core/views/install/logo.svg
and then exported to png with the width of 400px. However, 400px version didn't look as crisp as it could, so I increased it to around 1000px so that it looks perfectly crisp.

**Reviewers should focus on:**
N/A

**Screenshot**
current

This is how it was. The logo was blurry. Here's after change:

now

We're still using width="400" so the overall size didn't change, but it's crisp now.

**Necessity**

- [x] N/A

**Confirmed**

- [x] N/A

**Required changes:**

- [x] N/A

### CODE REVIEW

**Comment by soobinrho** [comment]
I read https://www.conventionalcommits.org/en/v1.0.0/#summary
and realized that my commit message wan't compliant, so
I changed the commit message.

**Comment by soobinrho** [comment]
I re-pushed my commit in order to add additional description in the commit message.

File: profile/README.md [hunk 1]
```diff
@@ -21,4 +21,4 @@ Flarum is scalable, performant and beautiful appealing to small to enterprise co
 - 🚲 Want to give Flarum a spin? Try the [demo](https://demo.flarum.site) (reset daily) or spin up your own community at [FreeFlarum](https://freeflarum.com).
 - 🧩 Curious which extensions exist? [Extiverse](https://extiverse.com) lists all public and premium Flarum extensions compatible with the latest release.
 - 💻 Wish to see Flarum communities in the wild? Visit [BuiltWithFlarum.com](https://builtwithflarum.com/) or see a global adoption rate at [BuiltWith](https://trends.builtwith.com/cms/Flarum).
-- 💰 Our funds are managed through our [Open Collective](https://opencollective.com/flarum), but we also take donations through [GitHub Sponsors](https://github.com/sponsors/flarum). More information on donating, like how we use the money, can be found at our [Donations page](https://flarum.org/donate).
\ No newline at end of file
+- 💰 Our funds are managed through our [Open Collective](https://opencollective.com/flarum), but we also take donations through [GitHub Sponsors](https://github.com/sponsors/flarum). More information on donating, like how we use the money, can be found at our [Donations page](https://flarum.org/donate).
```
Comment (soobinrho) [inline profile/README.md]: This is my editorconfig's doing. Nothing's being changed here except for trimming trailing white spaces.

**Comment by soobinrho** [comment]
@luceos Normally, I don't tag people, but I noticed that there's already a PR from four months ago in your .github repository. Assuming you guys aren't notified automatically by PR's in this repo, I had to tag you :)

**Comment by davwheat** [comment]
Is there any reason we can't just use the SVG version directly?

**Comment by soobinrho** [comment]
> Is there any reason we can't just use the SVG version directly?

Yes, that'd be the perfect solution actually. However, the svg file I found had an animation on it, which made it behave in unexpected ways:
https://github.com/flarum/framework/blob/main/framework/core/views/install/logo.svg

Now that I think about it, it'd be cool with the animation as well. Let's use the SVG version directly!

**Comment by davwheat** [comment]
@luceos Is it worth publicising our official brand SVG asset for this?

**Comment by luceos** [comment]
@davwheat I think that makes the most sense yes. Are you able to push that over? It might even be in the source of the site already..

**Comment by davwheat** [comment]
I can, just not tomorrow.

**Comment by soobinrho** [comment]
@davwheat As you suggested, I've changed the png to svg. 71a7fee 
It's perfectly crisp as it's a svg file, and its animation looks cool. Here's a preview:

output

Again, at this very moment, the logo that's being shown at the main repository is blurry. Since it's the first image we see when we go to https://github.com/flarum page, that's not good. Let me know if you'd like me to make any change to this commit to make it better :+1: 

Thank you guys always for everything you do for Flarum!

**Comment by soobinrho** [comment]
@davwheat I've found the .svg logo file in the framework repo and committed it to this PR. 71a7fee
https://github.com/flarum/framework/blob/main/framework/core/views/install/logo.svg

**Pull request merged**